### DISCUSSION TITLE
Help with the first example of authorization

### DISCUSSION THREAD

**Discussion by SerhiiZahuba** [authelia/authelia#7937] [General]
Can't find the right cofig. Please help. After installing authelia I have 4 links
public-ua.example.com
 traefik-ua.example.com
secure-ua.example.com
authelia-ua.example.com
. My grafana config on the screenshot. After authorization, the system does not redirect me anywhere, I think that my config is not correct. Please help. Thank you

![image](https://github.com/user-attachments/assets/91dbfadb-baa3-4c22-b053-3b51fff37d6e)
![image](https://github.com/user-attachments/assets/e2d1f7c3-e743-4244-bee0-7612e64513e8)

**Comment by james-d-elliott** [comment]
What do the logs show for grafana and authelia?

**Reply by SerhiiZahuba** [reply]
from authelia
![image](https://github.com/user-attachments/assets/e0f76f08-f52b-4246-9aed-e4589ff17fc9)

from grafana
![image](https://github.com/user-attachments/assets/60310924-a94d-4d18-9502-c20cfdd309a1)

**Reply by james-d-elliott** [reply]
Are these the logs when clicking the Login with Authelia link?

**Reply by SerhiiZahuba** [reply]
Yes

**Reply by james-d-elliott** [reply]
Can you try again with logs level set to `debug` in authelia? There should be authorization request and access request logs:

```yaml
log:
  level: debug
```

**Reply by SerhiiZahuba** [reply]
its all logs from start app with debug ( i use authelia in docker)

![image](https://github.com/user-attachments/assets/a4ab4877-b2eb-47b0-bffe-46e46d5dfef9)

**Reply by james-d-elliott** [reply]
You've configured the proxy to check authelia to see if users are authorized to access authelia, you'll need to adjust that before going further.  Specifically if you have the middleware applied to the authelia service or the /api/oidc/authz/* / /api/verify config on that server then remove it.

**Reply by SerhiiZahuba** [reply]
On the host with authelia, I only have docker from your example https://github.com/authelia/authelia/blob/master/examples/compose/local/docker-compose.yml is on this host.  From grafana host  https://www.authelia.com/integration/openid-connect/grafana

**Reply by james-d-elliott** [reply]
Show the authelia one, I don't believe it's possible if you're using that config.

**Reply by SerhiiZahuba** [reply]
https://gist.github.com/SerhiiZahuba/78e5f0b5fdf1de0eb44197f42d68e23e    docker compose
https://gist.github.com/SerhiiZahuba/d1ec5cab53b995b162f33c2463612d59  authelia-config

**Reply by james-d-elliott** [reply]
You've configured grafana to request OIDC auth from the whoami container, instead of authelia.

**Reply by SerhiiZahuba** [reply]
I'm going crazy now. That is, the problem is on the grafana side? I have 4 urls that will give me a script
public*
traefik*
secure*
authelia* 
where do I need to enter authorization from garafana?
I tried every option I could imagine

**Reply by james-d-elliott** [reply]
It needs to be the same URLs except instead of `secure-ua` it needs to be `authelia-ua`.

**Reply by SerhiiZahuba** [reply]
I'm sorry, but if I change the link to authelia-ua in grafana, I see a 404 error, there are no errors in the docker logs.

![image](https://github.com/user-attachments/assets/7756578c-e3ec-4c9d-b85c-ad611ca74e65)

**Reply by SerhiiZahuba** [reply]
I found my mistake, I did not enter the entire configuration. Now e-addressing is already working, there are other errors already on the grafana side, I will try to solve them. Thank you
